core/driver/sources/src/main/resources/reference.conf

```diff
@@ -161,34 +201,108 @@ probe {
       "https://cache-redirector.jetbrains.com/intellij-jbr/jbr-[major]-[platform]-x64-b[minor].tar.gz"
     ]
 
+    // The maximal number of retries that should be performed by resolvers when resolving a pattern to a resource.
     retries = 0
   }
 
   // `driver` is for `org.virtuslab.ideprobe.config.DriverConfig` - default values below
   driver {
+
+    // `driver.launch` contains settings that define how IDE should be started.
     launch {
+
+      // `driver.launch.command` is a string array transformed into the command used to launch the IDE (each string
+      // from the array is a single element of the command, spaces are inserted between the elements). By default,
```

Comment:
The process and its launch command is in the end prepared by `com.zaxxer.nuprocess.NuProcessBuilder`. In fact we pass `driver.launch.command` array (raw or with first element - "idea" command - expanded with `xvfb` / `headless` additions) to `NuProcessBuilder` constructor so that `NuProcessBuilder`'s field named `command` is equal to what we pass there.
And here is a javadoc about this field and how it is used:
```
a command, a list of strings which signifies the external program file to be invoked and its arguments, if any. Which string lists represent a valid operating system command is system-dependent. For example, it is common for each conceptual argument to be an element in this list, but there are operating systems where programs are expected to tokenize command line strings themselves - on such a system a Java implementation might require commands to contain exactly two elements.
```

Reply:
To be honest, I don't know the answer.